### DISCUSSION TITLE
Installing the CEA GUI

### DISCUSSION THREAD

**Discussion by KQEFEM** [architecture-building-systems/CityEnergyAnalyst#3283] [General]
Hi All, 
I am looking to install the CEA GUI (non-developer version) but nothing happens when I fill in my name and email address - see the attached image. 

I would like to initally avoid downloading the developer version until I have an understanding of what is going on in the package. 

If anyone could advise that would be idea! 

Regards
Kieran 

![image](https://user-images.githubusercontent.com/34337935/216601599-e746cdbd-fec6-4589-b436-50408163ebc3.png)

**Comment by lilaleu** [comment]
The issue has been fixed by now :)
